Tao_Current.txt

```diff
@@ -163,133 +187,159 @@ The IETF discussion list is unmoderated. This means that all can express their o
+
 3.6 Seeing Spots Before Your Eyes
 
 Some of the people at the IETF will have a little colored dot on their name tag. A few people have more than one. These dots identify people who are silly enough to volunteer to do a lot of extra work. The colors have the meanings shown here.
-Color 	Meaning
-Blue 	Working Group/BOF Chair
-Green 	Host group
-Red 	IAB member
-Yello 	IESG member
-Orange 	Nominating Committee member
-Purple 	IAOC
-Pink 	IRSG
-Teal 	RSE
+
+| Color  | Meaning                     |
+|--------|-----------------------------|
+| Blue   | Working Group/BOF Chair     |
+| Green  | Host group                  |
+| Red    | IAB member                  |
+| Yellow | IESG member                 |
+| Orange | Nominating Committee member |
+| Purple | IAOC                        |
+| Pink   | IRSG                        |
 
 (Members of the press wear orange-tinted badges.)
 
 Local hosts are the people who can answer questions about the terminal room, restaurants, and points of interest in the area.
 
 It is important that newcomers to the IETF not be afraid to strike up conversations with people who wear these dots. If the IAB and IESG members and Working Group and BOF chairs didn't want to talk to anybody, they wouldn't be wearing the dots in the first place. Note, however, that IETF meetings are usually intense times for Area Directors. Talking to an AD during an IETF meeting will often lead to a request to send her or him email about two weeks later. Also, when you start a hallway conversation with an Area Director (or even a WG chair, for that matter), it is often good to give them about 30 seconds of context for the discussion.
+
 3.7 Terminal Room
 
-One of the most important (depending on your point of view) things the host does is provide Internet access for the meeting attendees. In general, wireless connectivity is excellent in all the meeting rooms and most common areas, and the wired connectivity is provided in the terminal room. The people and companies that donate their equipment, services, and time are to be heartily congratulated and thanked.
+One of the most important (depending on your point of view) things the host does is provide unfettered Internet access for the meeting attendees. In general, wireless connectivity is excellent in all the meeting rooms and most common areas, and the wired connectivity is provided in the terminal room. The people and companies that donate their equipment, services, and time are to be heartily congratulated and thanked.
```

Comment:
I'm not sure how this document is using the term "host," but it doesn't match any of the concepts we have that I'm aware of. We typically have one or two Meeting Hosts, which are organizations that pay large sums of money to support a particular meeting. They are not involved in providing Internet connectivity. Internet connectivity is arranged by the NOC and typically relies on in-kind donations of circuits, or occasionally circuits we contract for.

Reply:
What about:

Every meeting the NOC provides unfettered Internet access for the meeting attendees. In general, wireless connectivity is excellent in all the meeting rooms and most common areas, and the wired connectivity is provided in the terminal room. The people and companies that donate their equipment, services, and time are to be heartily congratulated and thanked.

Reply:
Probably best to drop "unfettered" or borrow the words from the mtgvenue document.